pkg/endpoint/events.go

```diff
@@ -119,6 +120,84 @@ func (e *Endpoint) PolicyRevisionBumpEvent(rev uint64) {
 	}
 }
 
+/// EndpointNoTrackEvent contains all fields necessary to update the NOTRACK rules.
+type EndpointNoTrackEvent struct {
+	ep     *Endpoint
+	annoCB AnnotationsResolverCB
+}
+
+// Handle handles the NOTRACK rule update.
+func (ev *EndpointNoTrackEvent) Handle(res chan interface{}) {
+	var oldPort uint16
+
+	e := ev.ep
+
+	// If this endpoint is going away, remove all related NOTRACK rules.
```

Comment:
This comment doesn't seem to reflect your most recent update [here](https://github.com/cilium/cilium/pull/13805/files#r529046921).
```suggestion
	// If this endpoint is going away, nothing to do
```

Reply:
Done.